utils/gen_artifacts_toml.jl

```diff
@@ -20,16 +19,34 @@ for format ∈ formats
   global k
   for matrix ∈ eachrow(matrices)
     k += 1
-    url = "$ssmc_url/$format/$(matrix.group)/$(matrix.name).tar.gz"
+    if format == "mat"
+      sscm_file = "$(matrix.name).mat"
+      url = "$ssmc_url/$format/$(matrix.group)/$sscm_file"
+    else
+      sscm_file = "$(matrix.name).tar.gz"
+      url = "$ssmc_url/$format/$(matrix.group)/$sscm_file"
```

Comment:
You could set `url` after the `if` and use `ssmc_file = format == "mat" ? "$(matrix.name).mat" : "$(matrix.name).tar.gz"`